lasagne/tests/test_init.py

```diff
@@ -156,6 +190,10 @@ def test_he_normal_gain():
     assert -0.1 < sample.mean() < 0.1
     assert 0.9 < sample.std() < 1.1
 
+    sample = HeNormal(gain='relu').sample((100, 100))
+    assert -0.1 < sample.mean() < 0.1
+    assert 0.1 < sample.std() < 0.2
```

Comment:
Same here. But maybe we should use a shape that gives different results for Glorot and He, such as `(50,200)`. Adapt the tests if you're in the mood, otherwise I'll open a separate issue for that.